dist-persist/wbstack/src/Internal/ApiWbStackInit.php

```diff
@@ -138,4 +140,20 @@ public function getAllowedParams() {
             ],
         ];
     }
+
+    public function initMainPage() {
```

Comment:
suggestion: this can probably be a static method?

Reply:
I think it could be, probably all the others too. Personally I usually opt for conformity to the rest of legacy code first, maybe I should rethink that :thinking: happy to hear your opinion about it

Reply:
I would think making this static wouldn't break conformity with the rest of the code, but I understand that's kind of subjective. I'd prefer the extra information a static method gives me (i.e. it's stateless) over some slight inconsistency. But it's very minor, so if you prefer to keep it that way, I'm fine with that too.

Reply:
thanks for the input! makes sense and is interesting to think about.